### DISCUSSION TITLE
Discord Channel Access

### DISCUSSION THREAD

**Discussion by fcisio** [react-hookz/web#57] [General]
Hi! I landed here from a comment that was made on `react-use`.
I love this initiative, and I'm glad to see a successor for `react-use` which I love but is pretty much dying now.

There was mention of a Discord Channel on that comment. I was wondering if this channel still exists and if it open to the general public right now. (the invite on the comment was invalid)

Thanks

**Comment by xobotyi** [comment]
Is invite link outdated?
The server exists everyone are free to join.

**Reply by fcisio** [reply]
Could you give me the link to join the channel?

Maybe you guys could also add it to the readme

**Reply by xobotyi** [reply]
I'd did not yet since server is not completely configured, when I finish it I'll definitely add it to the readme.

Aah, it has 7 days expiration %)

The permanent one: https://discord.gg/Fjwphtu65f
